tests/e2e/cucumber/features/admin-settings/spaces.feature

```diff
@@ -9,11 +9,11 @@ Feature: spaces management
       | id    | role        |
       | Alice | Space Admin |
       | Brian | Space Admin |
-    When "Alice" logs in
-    And "Alice" creates the following project spaces using API
+    When "Alice" creates the following project spaces using API
       | name   | id     |
       | team A | team.a |
       | team B | team.b |
+    And "Alice" logs in
     When "Alice" opens the "admin-settings" app
     And "Alice" navigates to the project spaces management page
     When "Alice" updates the space "team.a" name to "developer team" using the context-menu
```

Comment:
```suggestion
    And "Alice" logs in
    And "Alice" opens the "admin-settings" app
    And "Alice" navigates to the project spaces management page
    And "Alice" updates the space "team.a" name to "developer team" using the context-menu
```

I don't understand why there is `when`

Reply:
The feature file of web are written in user-story. That's why this is different from feature file in API